detection_rules/misc.py

```diff
@@ -270,12 +270,16 @@ def load_current_package_version() -> str:
     return load_etc_dump('packages.yml')['package']['name']
 
 
+def get_default_config() -> Optional[Path]:
+    return next(Path(get_path()).glob('.detection-rules-cfg.*'), None)
```

Comment:
As we start to think about DaC, will we require users to use this specific file or are we expecting them to set environment variables as an alternative and skip this file altogether?

Reply:
Going to update it so that the config presence is not the defining factor.